lib/models/course.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:device_calendar/device_calendar.dart';
+
+/// This class models a course provided by Google Calendar.
+class Course {
+  String _summary, _description;
+  DateTime _start, _end;
+  String _location;
+
+  int duration() => _end.minute - _start.minute;
```

Comment:
Maybe rename to durationMinutes() or add doc comment specifying value returned is in minutes.

Reply:
resolved in 0a3a844a41233cf7585f116bc0e39a208cc86a22